modules/update-using-custom-machine-config-pools-mcp.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-mcp_{context}"]
+= Creating machine config pools to perform a canary rollout update
+
+The first task in performing this canary rollout update is to create one or more machine config pools (MCP). 
+
+. Create an MCP from a worker node.
+
+.. List the worker nodes in your cluster.
++
+[source,terminal]
+----
+$ oc get -l 'node-role.kubernetes.io/master!=' -o 'jsonpath={range .items[*]}{.metadata.name}{"\n"}{end}' nodes
+----
++
+.Example output
++
+[source,terminal]
+----
+ci-ln-pwnll6b-f76d1-s8t9n-worker-a-s75z4
+ci-ln-pwnll6b-f76d1-s8t9n-worker-b-dglj2
+ci-ln-pwnll6b-f76d1-s8t9n-worker-c-lldbm
+----
+
+.. For the nodes you want to delay, add a custom label to the node:
++
+[source,terminal]
+----
+$ oc label node <node name> node-role.kubernetes.io/<custom-label>=
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc label node ci-ln-0qv1yp2-f76d1-kl2tq-worker-a-j2ssz node-role.kubernetes.io/workerpool-canary=
```

Comment:
Should there be an equal sign at the end of this?

Reply:
See above